uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/leave-request-api_test.js

```diff
@@ -169,6 +166,33 @@ define([
       });
     });
 
+    describe('balanceChangeByAbsenceType() with error from server', function () {
```

Comment:
You already have a `describe('balanceChangeByAbsenceType')` section
```js
describe('balanceChangeByAbsenceType()', function () {


  describe('when the server returns an error', function () { // or whatever description is better

  });
});
```
this applies for every other test where you have a new "with error from server" section